spec/models/season/phase_switcher_spec.rb

```diff
@@ -53,9 +53,7 @@
 
     it 'fails silently when it receives a non-whitelisted phase' do
       phase = 'bad_intentions'
```

Comment:
Could you add these two lines right before the `expect {} ` block?

```ruby
# Just to make sure we have our season in the test DB.
# It creates one if it didn't already exist though some other factory:
Season.current 
allow_any_instance_of(Season).to receive(:save).and_raise RuntimeError
```

If the test outputs a RuntimeError, something is updating the season where it shouldn't get an update. If the date error prevails, it's the rounding.